pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.Ericsson</groupId>
+        <artifactId>eiffel-remrem-parent</artifactId>
+        <version>0.0.2</version>
+    </parent>
+    <properties>
+        <eiffel-shared.version>0.3.6</eiffel-shared.version>
+        <eiffel-semantics.version>0.4.5</eiffel-semantics.version>
+        <eiffel-remrem-generate.version>0.10.2</eiffel-remrem-generate.version>
+    </properties>
+    <artifactId>eiffel-remrem-generate</artifactId>
+    <version>${eiffel-remrem-generate.version}</version>
+    <packaging>pom</packaging>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.github.Ericsson</groupId>
+                <artifactId>eiffel-remrem-shared</artifactId>
+                <version>${eiffel-shared.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.Ericsson</groupId>
+                <artifactId>eiffel-remrem-semantics</artifactId>
+                <version>${eiffel-semantics.version}</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+    <modules>
+        <module>cli</module>
+        <module>service</module>
+    </modules>
+    <repositories>
+        <repository>
+            <id>jitpack.io</id>
+            <name>Jitpack Repository</name>
+            <url>https://jitpack.io/</url>
+        </repository>
```

Comment:
add this also in repositories
maven { url 'https://repo.spring.io/libs-milestone' }